cms/djangoapps/contentstore/views/course.py

```diff
@@ -372,19 +373,25 @@ def format_course_for_view(course):
             'rerun_link': _get_rerun_link_for_item(course.id),
             'org': course.display_org_with_default,
             'number': course.display_number_with_default,
-            'run': course.location.run
+            'run': course.location.run,
+            'license': course.license,
+            'licenseable': course.licenseable
         }
 
+        return fields
+
     def format_in_process_course_view(uca):
         """
         Return a dict of the data which the view requires for each unsucceeded course
         """
-        return {
+        fields = {
```

Comment:
Ah, good one. I missed that change, will revert it. The license and licensable attributes were optional before.